lib/config/presets/internal/preview.ts

```diff
@@ -2,13 +2,13 @@ import type { Preset } from '../types';
 
 export const presets: Record<string, Preset> = {
   dockerCompose: {
-    description: 'Enable docker compose image updating',
+    description: 'Enable Docker Compose image updating',
```

Comment:
Are we happy with the `chore:` prefix for the PR title, or should I use `fix:` instead?

Reply:
I think it's more a docs scope.